cypress/integration/visitorCanSeeSideArticles.js

```diff
@@ -0,0 +1,25 @@
+describe("Visitor can see side articles", () => {
+  beforeEach(() => {
+    cy.server();
+  });
+  
+  it("successfully", () => {
+    cy.route({
+      method: "GET",
+      url: "http://localhost:3000/api/v1/articles",
+      response: "fixture:side_articles_shown.json"
+    });
+    cy.visit("/");
+    cy.get("#side-articles-div-2").should("contain", "TestBody2");
```

Comment:
add an assertion that we see 3 article elements in the sidebar div

Reply:
four asserted in this PR as it corresponds to user story.